### PR TITLE
Update collectd-openstack with new config options

### PR DESCRIPTION
**Pull request by rmfitzpatrick**
~Requires https://github.com/signalfx/collectd-openstack/pull/9, and the plugin version will need to be changed once it lands.~

These changes adopt the four new options for the Openstack plugin and ensure that floats are supported in collectd configs.

### CODE REVIEW

**Review by emaderer — APPROVED** [review]

File: docs/monitors/collectd-openstack.md [hunk 1]
```diff
@@ -43,15 +43,20 @@ monitors:
   authURL: "http://192.168.11.111/identity/v3"
   username: "admin"
   password: "secret"
+  requestBatchSize: 10
+  novaListServersSearchOpts:
+    all_tenants: "TRUE"
+    status: "ACTIVE"
 ```
-### Example config using skipVerify
+### Example config using skipVerify and disabling querying server metrics:
```
Comment (theletterf) [inline docs/monitors/collectd-openstack.md:51]:
```suggestion
### Example config using skipVerify and disabling querying server metrics
```

File: docs/monitors/collectd-openstack.md [hunk 2]
```diff
@@ -81,6 +86,10 @@ Configuration](../monitor-config.md#common-configuration).**
 | `regionName` | no | `string` | The region name for URL discovery, defaults to the first region if multiple regions are available. |
 | `userDomainID` | no | `string` | The user domain id (**default**:"default") |
 | `skipVerify` | no | `bool` | Skip SSL certificate validation (**default:** `false`) |
+| `httpTimeout` | no | `float64` | The http client timeout in seconds for all requests (**default:** `0`) |
+| `requestBatchSize` | no | `integer` | The maximum number of concurrent requests for each metric class (**default:** `5`) |
+| `queryServerMetrics` | no | `bool` | Whether to query server metrics (useful to disable for TripleO Undercloud) (**default:** `true`) |
+| `novaListServersSearchOpts` | no | `map of strings` | Optional search_opts mapping for collectd-openstack Nova client servers.list(search_opts=novaListServerSearchOpts). For more information see https://docs.openstack.org/api-ref/compute/#list-servers. |
```
Comment (theletterf) [inline docs/monitors/collectd-openstack.md:92]:
```suggestion
| `httpTimeout` | no | `float64` | The HTTP client timeout in seconds for all requests (**default:** `0`) |
| `requestBatchSize` | no | `integer` | The maximum number of concurrent requests for each metric class (**default:** `5`) |
| `queryServerMetrics` | no | `bool` | Whether to query server metrics (useful to disable for TripleO Undercloud) (**default:** `true`) |
| `novaListServersSearchOpts` | no | `map of strings` | Optional search_opts mapping for collectd-openstack Nova client servers.list(search_opts=novaListServerSearchOpts). For more information see https://docs.openstack.org/api-ref/compute/#list-servers. |
```

File: pkg/monitors/collectd/openstack/metadata.yaml [hunk 3]
```diff
@@ -38,15 +38,20 @@ monitors:
       authURL: "http://192.168.11.111/identity/v3"
       username: "admin"
       password: "secret"
+      requestBatchSize: 10
+      novaListServersSearchOpts:
+        all_tenants: "TRUE"
+        status: "ACTIVE"
     ```
-    ### Example config using skipVerify
+    ### Example config using skipVerify and disabling querying server metrics:
```
Comment (theletterf) [inline pkg/monitors/collectd/openstack/metadata.yaml:46]:
```suggestion
    ### Example config using skipVerify and disabling querying server metrics
```

**Review by theletterf — APPROVED** [review]
Just a few nits.

**Pull request merged**